src/Handpose/index.js

```diff
@@ -61,19 +62,42 @@ class Handpose {
   async loadModel() {
     const pipeline = handPoseDetection.SupportedModels.MediaPipeHands;
     //filter out model config options
-    const modelConfig = {
-      maxHands: this.config?.maxHands ?? 2,
-      runtime: this.config?.runtime ?? "mediapipe",
-      modelType: this.config?.modelType ?? "full",
-      solutionPath:
-        this.config?.solutionPath ??
-        "https://cdn.jsdelivr.net/npm/@mediapipe/hands",
-      detectorModelUrl: this.config?.detectorModelUrl,
-      landmarkModelUrl: this.config?.landmarkModelUrl,
-    };
-    this.runtimeConfig = {
-      flipHorizontal: this.config?.flipHorizontal ?? false,
-    };
+    const modelConfig = handleOptions(this.config, {
```

Comment:
The options object itself could be defined as a `static` property or outside of the model class.  It doesn't depend of `this` because of how you've designed the conditionals as a function of `config`.  The sort of structured information that you have here is golden for scraping into a documentation site.  We would want to have some sort of system for that where we know that the options config is `Handpose.optionSchema` or whatever.  Not sure what we would do with those functions in the documentation though.